packages/openassistants-react/tsconfig.json

```diff
@@ -6,7 +6,7 @@
     "lib": ["dom", "esnext"],
     "importHelpers": true,
     // output .d.ts declaration files for consumers
-    "declaration": true,
+    "declaration": false,
```

Comment:
I'm getting: 

(typescript) Error: .../openassistants/packages/openassistants-react/src/components/ui/button.tsx(7,7): semantic error TS2742: The inferred type of 'buttonVariants' cannot be named without a reference to 'class-variance-authority/node_modules/clsx'. This is likely not portable. A type annotation is necessary.

Reply:
@colemccracken did you get this before? Any ideas how to fix? Do we need declartion outputs?

Reply:
github.com/joe-bell/cva/issues/163

Ah, they don't export types. I don't particularly like the solution I've committed (copying their types). Maybe we should look for an alternative.

Reply:
Seems fine. Sounded like he intended to fix in 1.0 which is currently in beta so we can probably remove on upgrade